libs/cardano-data/CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Version history for `cardano-data`
 
-## 1.1.0.1
+## 1.2.0.0
```

Comment:
Addition of a module is not a breaking change:
```suggestion
## 1.1.2.0
```

Reply:
Oh, sorry! Noted! :memo:

Reply:
@lehins do you mean
```suggestion
## 1.1.1.0
```
instead of the double bump from `0` to `2` directly in the `minor` number?

Reply:
Yes, good catch, I skipped a version somehow in my suggestion :facepalm: